adr/0003-polymorphic-slot-definitions.md

```diff
@@ -15,7 +15,7 @@ Accepted
 Components can currently define slots in two ways:
 
 1. by specifying a component class (or class name string)
-1. by providing a proc (i.e. lambda) that either returns HTML or a component instance.
+1. by providing a proc (that's lambda) that either returns HTML or a component instance.
```

Comment:
I think this may need some manual updating since it now reads strangely

Reply:
```suggestion
1. by providing a proc that either returns HTML or a component instance.
```